test/function-overrides/app.module.ts

```diff
@@ -0,0 +1,38 @@
+import { ExecutionContext, Module } from '@nestjs/common';
+import { APP_GUARD } from '@nestjs/core';
+import { ThrottlerGuard, ThrottlerModule, seconds } from '../../src';
+import { FunctionOverridesThrottlerController } from './funciton-overrides-throttler.controller';
+import md5 = require('md5');
+import assert = require('assert');
+
+@Module({
+  imports: [
+    ThrottlerModule.forRoot([
+      {
+        name: 'default',
+        ttl: seconds(3),
+        limit: 2,
+      },
+      {
+        name: 'custom',
+        ttl: seconds(3),
+        limit: 2,
+        getTracker: () => 'customTrackerString',
+        generateKey: (context: ExecutionContext, trackerString: string, throttlerName: string) => {
+          // check if tracker string is passed correctly
+          assert(trackerString === 'customTrackerString');
```

Comment:
I hope this line is ok in for the test. I couldn't think of a much better place to test if the `getTracker` method is actually overridden correctly.

Reply:
Yeah, this is fine. I might enhance the tests later, but this is good for the time being. Thanks for adding them